docs/site/_includes/getting_started/openstack/partials/config.yml.standard.ee.inc

```diff
@@ -86,6 +86,11 @@ provider:
   # [<en>] You might consider changing this.
   # [<ru>] Возможно, захотите изменить.
   region: RegionOne
+  # [<en>] You might need to specify tenantName instead of tenantID.
+  # [<en>] https://deckhouse.io/documentation/v1/modules/030-cloud-provider-openstack/cluster_configuration.html#openstackclusterconfiguration-provider-tenantname
+  # [<ru>] Возможно вам нужно указать tenantName вместо tenantID.
+  # [<ru>] https://deckhouse.ru/documentation/v1/modules/030-cloud-provider-openstack/cluster_configuration.html#openstackclusterconfiguration-provider-tenantname
+  #  tenantName: *!CHANGE_PROJECT_NAME*
```

Comment:
```suggestion
  # [<en>] You might need to specify tenantName instead of tenantID.
  # [<en>] https://deckhouse.io/documentation/v1/modules/030-cloud-provider-openstack/cluster_configuration.html#openstackclusterconfiguration-provider-tenantname
  # [<ru>] Возможно вам нужно указать tenantName вместо tenantID.
  # [<ru>] https://deckhouse.ru/documentation/v1/modules/030-cloud-provider-openstack/cluster_configuration.html#openstackclusterconfiguration-provider-tenantname
  # tenantName: PROJECT_NAME
```